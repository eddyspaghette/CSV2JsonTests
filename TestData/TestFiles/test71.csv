1|100,200,300,400,500
2,102,201,301,401,501
3,104,202,302,402,502
4,106,203,303,403,503
23,144,222,322,422,522
,146,223,323,423,523
7,112,206,306,406,506
8,,207,307,407,507
9,116,,308,408,508
10,118,209,309,409,509
11,120,210,310,410,
23,144,222,322,422,522
24,146,223,323,423,523
14,126,213,313,413,513
15,128,214,314,414,514
16,130,,315,415,515
17,132,216,316,416,516
18,134,217,317,417,517
19,136,218,318,418,518
20,138,219,319,419,519
21,140,220,320,420,520
22,142,221,321,421,521
23,144,222,322,422,522
24,146,223,323,423,523
