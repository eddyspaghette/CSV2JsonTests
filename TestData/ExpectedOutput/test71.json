[
 {
   "1|100": 2,
   "200": 102,
   "300": 201,
   "400": 301,
   "500": 401,
   "FIELD6": 501
 },
 {
   "1|100": 3,
   "200": 104,
   "300": 202,
   "400": 302,
   "500": 402,
   "FIELD6": 502
 },
 {
   "1|100": 4,
   "200": 106,
   "300": 203,
   "400": 303,
   "500": 403,
   "FIELD6": 503
 },
 {
   "1|100": 23,
   "200": 144,
   "300": 222,
   "400": 322,
   "500": 422,
   "FIELD6": 522
 },
 {
   "1|100": null,
   "200": 146,
   "300": 223,
   "400": 323,
   "500": 423,
   "FIELD6": 523
 },
 {
   "1|100": 7,
   "200": 112,
   "300": 206,
   "400": 306,
   "500": 406,
   "FIELD6": 506
 },
 {
   "1|100": 8,
   "200": null,
   "300": 207,
   "400": 307,
   "500": 407,
   "FIELD6": 507
 },
 {
   "1|100": 9,
   "200": 116,
   "300": null,
   "400": 308,
   "500": 408,
   "FIELD6": 508
 },
 {
   "1|100": 10,
   "200": 118,
   "300": 209,
   "400": 309,
   "500": 409,
   "FIELD6": 509
 },
 {
   "1|100": 11,
   "200": 120,
   "300": 210,
   "400": 310,
   "500": 410,
   "FIELD6": null
 },
 {
   "1|100": 23,
   "200": 144,
   "300": 222,
   "400": 322,
   "500": 422,
   "FIELD6": 522
 },
 {
   "1|100": 24,
   "200": 146,
   "300": 223,
   "400": 323,
   "500": 423,
   "FIELD6": 523
 },
 {
   "1|100": 14,
   "200": 126,
   "300": 213,
   "400": 313,
   "500": 413,
   "FIELD6": 513
 },
 {
   "1|100": 15,
   "200": 128,
   "300": 214,
   "400": 314,
   "500": 414,
   "FIELD6": 514
 },
 {
   "1|100": 16,
   "200": 130,
   "300": null,
   "400": 315,
   "500": 415,
   "FIELD6": 515
 },
 {
   "1|100": 17,
   "200": 132,
   "300": 216,
   "400": 316,
   "500": 416,
   "FIELD6": 516
 },
 {
   "1|100": 18,
   "200": 134,
   "300": 217,
   "400": 317,
   "500": 417,
   "FIELD6": 517
 },
 {
   "1|100": 19,
   "200": 136,
   "300": 218,
   "400": 318,
   "500": 418,
   "FIELD6": 518
 },
 {
   "1|100": 20,
   "200": 138,
   "300": 219,
   "400": 319,
   "500": 419,
   "FIELD6": 519
 },
 {
   "1|100": 21,
   "200": 140,
   "300": 220,
   "400": 320,
   "500": 420,
   "FIELD6": 520
 },
 {
   "1|100": 22,
   "200": 142,
   "300": 221,
   "400": 321,
   "500": 421,
   "FIELD6": 521
 },
 {
   "1|100": 23,
   "200": 144,
   "300": 222,
   "400": 322,
   "500": 422,
   "FIELD6": 522
 },
 {
   "1|100": 24,
   "200": 146,
   "300": 223,
   "400": 323,
   "500": 423,
   "FIELD6": 523
 }
]